.rubocop.yml

```diff
@@ -11,6 +11,13 @@ AllCops:
     - 'script/**/*'
     - 'vendor/**/*'
 
+Metrics/BlockLength:
```

Comment:
why not fixing this offenses instead? :thinking: and even if you don't fix them... shouldn't they be in the todo file? od don't we plan to solve them? :thinking:

Reply:
How would you split that method?

I really dislike the metrics part. Spliting functions around because there is a magic number considered good is not something I believe it makes sense. It creates artificial jumping when reading code. It is not even about reusability (which the tool can't detect).

Because it is on the `test_helper` and all the violations make sense (long urls, long blocks generating xml), I thought it made sense to make an exception.

Reply:
the funny part: you can outsource this code to file in views/...xml.builder - and no tool will complain it's too long, because everyone understands that long XML require long builder code. But if it's in a helper, it obviously need to be split.

Reply:
Exclude it then locally, it is not worthwhile to make it unreadable to make Rubocop happy :wink:

Reply:
I have disabled the check in the method itself.